agents/postgres/pgstatmonitor/models.go

```diff
@@ -0,0 +1,99 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgstatmonitor
+
+import (
+	"fmt"
+	"time"
+
+	pq_types "github.com/mc2soft/pq-types"
```

Comment:
We should not use that library directly. If we don't have required code somewhere already (please search), let's just copy the part we need

Reply:
I searched and found that StringArray is also contained in pg package. So now I using it from here.